### ISSUE TITLE
[Bug]: pymilvus v2.0.0rc8 search slowly

### ISSUE THREAD

**Issue by ownlu** [milvus-io/milvus#13780]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):pymilvus v2.0.0rc8
- OS(Ubuntu or CentOS): CentOS
- CPU/Memory: 200g
- GPU: Tesla P4
- Others:  docker-compose standalone
```

### Current Behavior

search slowly,cost longer than 60s.

### Expected Behavior

want to find out reason 

### Steps To Reproduce

```markdown
1、docker-compose start milvus-standalone
2、run hello-milvus
3、search cost about 0.5s
4、after two days,search cost about 20s
5、after two days,search cost about 60s
6、after two days,search keep searching,no response
```

### Anything else?

Here's the log after start milvus:

milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:33.747 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":493}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:34.301 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":551}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:35.137 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":832}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:35.668 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":528}
milvus-standalone | {"level":"DEBUG","time":"2021/12/20 08:53:36.569 +00:00","caller":"session_util.go:277","message":"SessionUtil GetSessions","prefix":"IndexNode","resp":{"header":{"cluster_id":14841639068965178418,"member_id":10276657743932975437,"revision":422403,"raft_term":5},"kvs":[{"key":"YnktZGV2L21ldGEvc2Vzc2lvbi9JbmRleE5vZGUtMjk=","create_revision":421766,"mod_revision":421766,"version":1,"value":"eyJTZXJ2ZXJJRCI6MjksIlNlcnZlck5hbWUiOiJJbmRleE5vZGUiLCJBZGRyZXNzIjoiMTcyLjE5LjAuNDoyMTEyMSJ9","lease":7587859309586971952}],"count":1}}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:36.658 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":987}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:36.893 +00:00","caller":"server.go:420","message":"Time tick lag behind for more than 1 minutes","channel":"by-dev-rootcoord-dml_3_429801518499364865v1","tt":"2021/12/20 07:20:50.874 +00:00"}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:36.963 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":302}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:37.424 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":459}
milvus-standalone | {"level":"DEBUG","time":"2021/12/20 08:53:37.547 +00:00","caller":"index_coord.go:628","message":"IndexCoord recycleUnusedIndexFiles","Need recycle tasks num":0}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:37.965 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":539}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:38.453 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":485}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:38.791 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":336}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:39.291 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":497}
milvus-standalone | {"level":"DEBUG","time":"2021/12/20 08:53:39.586 +00:00","caller":"session_util.go:277","message":"SessionUtil GetSessions","prefix":"IndexNode","resp":{"header":{"cluster_id":14841639068965178418,"member_id":10276657743932975437,"revision":422406,"raft_term":5},"kvs":[{"key":"YnktZGV2L21ldGEvc2Vzc2lvbi9JbmRleE5vZGUtMjk=","create_revision":421766,"mod_revision":421766,"version":1,"value":"eyJTZXJ2ZXJJRCI6MjksIlNlcnZlck5hbWUiOiJJbmRleE5vZGUiLCJBZGRyZXNzIjoiMTcyLjE5LjAuNDoyMTEyMSJ9","lease":7587859309586971952}],"count":1}}

**Comment by yanliang567** [comment]
it sounds like a known issue in rc8, which is going to fix in next release.
@ownlu what operations did you do during the 2-days waiting?
/assign @ownlu 
/unassign

**Comment by ownlu** [comment]
> 这听起来像是 rc8 中的一个已知问题，它将在下一个版本中修复。 @ownlu在这两天的等待中你做了什么操作？ /assign @ownlu /unassign

Created another collection（“collection2” 512dim) with the same way.Inserted 2 vectors in this collection, but search slowy after two days,then I delete it. Then the "Hello_milvus" collection search slowy as well.

**Comment by ownlu** [comment]
> it sounds like a known issue in rc8, which is going to fix in next release. @ownlu what operations did you do during the 2-days waiting? /assign @ownlu /unassign

Created another collection（“collection2” 512dim) with the same way.Inserted 2 vectors in this collection, but search slowy after two days,then I delete it. Then the "Hello_milvus" collection search slowy as well.

**Comment by yanliang567** [comment]
@zhuwenxing is it the same issue you met before?
/assign @zhuwenxing

**Comment by ownlu** [comment]
> @zhuwenxing is it the same issue you met before? /assign @zhuwenxing

Yes. 
rm -rf volumes, restart the image, rebuild collection several times. Issues always come up.

I changed from v2.0.0-rc8 to v2.0.0-rc7 today.Until now the problem has not appeared.

**Comment by xiaofan-luan** [comment]
hi @ownlu , we've already released 2.0-preGA and that should probably solved your problem
